matrix-vis.js

```diff
@@ -105,11 +105,24 @@ function initGraph() {
     network = new vis.Network(container, data, options);
     // add event listeners
     network.on('select', function(params) {
-        var text = "";
+        var Type = " ";
+        var Sender = " ";
+        var m = " ";
```

Comment:
This seems to exist only to be used as a temporary variable. You don't need it. In-line your assignments instead:
```
t3 = "Action : " + nodes.get(params.nodes[0]).blob.content.membership;
```